spec/lib/account_deleter_spec.rb

```diff
@@ -111,7 +111,7 @@
     before do
       @account_deletion.person = bob.person
     end
-    it 'removes all standard person associaltions' do
+    it 'removes all standard person associations' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.